internal/model/manifest.go

```diff
@@ -109,6 +109,16 @@ func (m Manifest) WithDeployTarget(t TargetSpec) Manifest {
 	return m
 }
 
+func (m Manifest) IsImageDeployed(iTarget ImageTarget) bool {
+	id := iTarget.ID()
+	for _, depID := range m.DeployTarget().DependencyIDs() {
```

Comment:
checking my understanding: only images that we care about deploying show up as deps of a DeployTarget (otherwise, they'd be deps of an image target representing the thing we actually want to deploy)?

sorta ambiguous function name, could maybe mean "is the image live in the cluster?" -- consider clarifying with a comment or renaming the func

Reply:
hmmmm...this "isDeployed" terminology is already used a bunch of other places in the codebase. if we decide to change it here, let's do it everywhere

"Live" seems like a much worse name, because it's likely to get confused with live update

Reply:
oh oh, I was NOT suggesting that we use "live" -- i was saying that "isDeployed" is confusing btwn "this is an image that we deploy, as opposed to an image that we don't" and "this is an image that is deployed to the cluster, as opposed to one that has not yet been sent to the cluster"

but sure, I don't have strong feelings